modules/related-posts/jetpack-related-posts.php

```diff
@@ -674,6 +674,14 @@ protected function _get_related_post_data_for_post( $post_id, $position, $origin
 			'title' => $this->_to_utf8( $this->_get_title( $post->post_title, $post->post_content ) ),
 			'date' => get_the_date( '', $post->ID ),
 			'format' => get_post_format( $post->ID ),
+			/**
+			 * Filters the rel attribute for the Related Posts' links.
+			 *
+			 * @since 3.7.0
+			 *
+			 * @param bool true Should the link rel attribute for Related Posts' links be dislpayed? Default is yes (true).
+			 */
+			'rel' => apply_filters( 'jetpack_relatedposts_filter_post_link_rel_enabled', true ),
```

Comment:
Re `rel` attributes , I can see some may want to prefetch related posts if they think there's a high chance that someone would want to click on a link:
http://www.w3schools.com/tags/att_a_rel.asp

This also plays into passing a post id as an optional argument. If I know this one post of mine is always super popular I might want to remove nofollow and add a prefetch to it every time that post gets related to something else.